pkgs/development/python-modules/astropy/default.nix

```diff
@@ -37,10 +38,13 @@ buildPythonPackage rec {
     hash = "sha256-5cb0XZEcMKy41VbH+O2ZSuxxsQjmHu5QZ/AK8eTjYTg=";
   };
 
-  postPatch = ''
-    substituteInPlace pyproject.toml \
-      --replace-fail "numpy>=2.0.0"  "numpy"
-  '';
```

Comment:
Not sure to understand why this removal is part of this change.

Reply:
> Not sure to understand why this removal is part of this change.

You are right! It was a cherry-pick + merge conflict mistake.